packages/scene-composer/src/components/toolbars/common/ItemContainer.tsx

```diff
@@ -0,0 +1,150 @@
+import { isEmpty } from 'lodash';
+import React, { ReactNode, useCallback, useState } from 'react';
+
+import { getGlobalSettings } from '../../../common/GlobalSettings';
+
+import {
+  Icon,
+  SubMenuIconContainer,
+  ToolbarItemContainer,
+  ToolbarItemIcon,
+  ToolbarItemMenu,
+  ToolbarItemText,
+} from './styledComponents';
+import { ToolbarItemOptions, ToolbarItemOrientation, ToolbarItemType } from './types';
+
+interface ToolbarItemContainerProps {
```

Comment:
Arrange props in an order based on mandatory vs optional.

Reply:
ok